packages/data-contract/index.js

```diff
@@ -0,0 +1,10 @@
+const { deployContract } = require("./src/helpers/deploy-contract");
+const { initClient } = require("./src/helpers/init-client")
+const dotenv = require('dotenv')
+dotenv.config()
+function main() {
+    const client = initClient();
+    deployContract(client).catch(console.error);
+}
+
+main()
```

Comment:
I think catch should be present on line 10, while function itself marked `async`